TypeCobol/Compiler/Types/TypedefType.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler.Symbols;
+
+namespace TypeCobol.Compiler.Types
+{
+    /// <summary>
+    /// A TypeDef type.
+    /// </summary>
+    public class TypedefType : Type
+    {
+        /// <summary>
+        /// Empty constructor.
+        /// <param name="symbol">Typedef's symbol</param>
+        /// </summary>
+        public TypedefType(TypedefSymbol symbol)
+            : base(Tags.Typedef)
+        {
+            this.Symbol = symbol;
+        }
+
+        /// <summary>
+        /// Type Constructor
+        /// </summary>
+        /// <param name="symbol">Typedef's symbol</param>
+        /// <param name="targetType">Typedef's type</param>
+        public TypedefType(TypedefSymbol symbol, Type targetType)
+            : base(Tags.Typedef)
+        {
+            System.Diagnostics.Debug.Assert(targetType != null);
+            TargetType = targetType;
+            this.Symbol = symbol;
+        }
+
+        /// <summary>
+        /// The target type of the Typedef
```

Comment:
Can you add an example in the comment?
With:
```cobol
01 Type1 typedef strict private pic X.
01 Type2 typedef strict private.
   05 Var1 pic X.
```

Reply:
ok